src/frontend/Player/createPlayer.spec.ts

```diff
@@ -46,6 +46,7 @@ describe('createPlayer', () => {
       ref,
       dispatchPlayerTimeUpdate,
       video,
+      undefined,
```

Comment:
> starts the player at 50% for a student to prevent volume too high on startup

I really don't like this behavior. Why not still using 100% ? It's easier to change the system volume than the player one.

Reply:
Your call, I'm quite used to start watching a video with a trash audio mix and the volume is too high so I have to manually reduce the volume (on the player or on the system) and I prefer having to increase the volume when it is too low than paying the price with my ears.
But if you prefer I can remove this for the student and use this only to mute the player for the teacher when he watch his live return.

Reply:
Yes keep 100% please.

Reply:
I checked and I never made this change.
I just lied in my commit message 😲